man/ctags-lang-verilog.7.rst.in

```diff
@@ -0,0 +1,85 @@
+.. _ctags_lang-verilog(7):
+
+======================================================================
+ctags-lang-verilog
+======================================================================
+
+:Version: @VERSION@
+:Manual group: Universal-ctags
+:Manual section: 7
+
+SYNOPSIS
+--------
+|	**@CTAGS_NAME_EXECUTABLE@** ... [--fields-Verilog=+{parameter}] ...
+|	**@CTAGS_NAME_EXECUTABLE@** ... [--kinds-systemverilog=+Q] [--fields-SystemVerilog=+{parameter}] ...
+|	**@CTAGS_NAME_EXECUTABLE@** --list-kinds-full={Verilog|SystemVerilog}
+|	**@CTAGS_NAME_EXECUTABLE@** --list-fields={Verilog|SystemVerilog}
+
+    +---------------+---------------+-------------------+
+    | Language      | Language ID   | File Mapping      |
+    +===============+===============+===================+
+    | Verilog       | Verilog       | .v                |
+    +---------------+---------------+-------------------+
+    | SystemVerilog | SystemVerilog | .sv, .svh, svi    |
+    +---------------+---------------+-------------------+
+
+DESCRIPTION
+-----------
+This man page describes about the Verilog/SystemVerilog parser for Universal-ctags.
+
+It assumes the input file is written in the correct grammer.  Otherwise output of
+ctags is undefined.
+
+``parameter`` field
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the field ``parameter`` is enabled, a tagfield ``parameter:`` is added on a parameter whose
+value can be overridden on an instantiated module, interface, or program.
+This is useful for a editor plugin or extension to enable auto-instatiation of modules with
+parameters which can be overridden.
+
+.. code-block:: console
+
+    $ ctags ... --fields-Verilog=+{parameter} ...
+    $ ctags ... --fields-SystemVerilog=+{parameter} ...
+
+On the following source code tagfields ``parameter:`` are added on
+parameter ``P*``, not on ``L*``.  Note that ``L4`` and ``L6`` is declared by
+``parameter`` statement, but tagfields ``parameter:`` are not added,
+because they cannot be overridden.
+
```

Comment:
When showing an example, could you follow the form I used in ctags-lang-python.7.rst.in?
Please, show not only input but also its tags output with the command line for emitting the output.

```
"input.sv"
.. code-block:: systemverilog
   ...

"output.tags"
with "--options=NONE -o - --fields-SystemVerilog=+{parameter} input.sv"
.. code-block:: tags
   ...
```

I'm thinking about generating a test case from the man page.